packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -41,35 +44,59 @@ Please make sure you are running "shorebird init" from the root of your Flutter
       return ExitCode.software.code;
     }
 
-    late final bool shorebirdYamlExists;
+    late final ShorebirdYaml? shorebirdYaml;
```

Comment:
Does this need to be late since it's already optional?

Reply:
Yup good call it's not needed 👍